fmriprep/cli/_version.py

```diff
@@ -0,0 +1,78 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""Version CLI helpers."""
```

Comment:
I don't like giving this a name like `_version.py`. Why not just `version` or `version_check`?